### ISSUE TITLE
Prophet breaking on holidays 0.25

### ISSUE THREAD

**Issue by dhuang** [facebook/prophet#2430]
Hello, we're seeing errors just trying to import prophet with the 0.25 version of holidays that was just released.

```
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.10/site-packages/prophet/__init__.py", line 7, in <module>
    from prophet.forecaster import Prophet
  File "/usr/local/lib/python3.10/site-packages/prophet/forecaster.py", line 19, in <module>
    from prophet.make_holidays import get_holiday_names, make_holidays_df
  File "/usr/local/lib/python3.10/site-packages/prophet/make_holidays.py", line 14, in <module>
    import prophet.hdays as hdays_part2
  File "/usr/local/lib/python3.10/site-packages/prophet/hdays.py", line 779, in <module>
    class TU(Turkey):
  File "/usr/local/lib/python3.10/site-packages/holidays/registry.py", line 178, in __init__
    super().__init__(*args, **kwargs)
TypeError: object.__init__() takes exactly one argument (the instance to initialize)
```

Please advise! We've pinned it for now.

**Comment by pjbhaumik** [comment]
> Hello, we're seeing errors just trying to import prophet with the 0.25 version of holidays that was just released.
> 
> ```
> Traceback (most recent call last):
>   File "<string>", line 1, in <module>
>   File "/usr/local/lib/python3.10/site-packages/prophet/__init__.py", line 7, in <module>
>     from prophet.forecaster import Prophet
>   File "/usr/local/lib/python3.10/site-packages/prophet/forecaster.py", line 19, in <module>
>     from prophet.make_holidays import get_holiday_names, make_holidays_df
>   File "/usr/local/lib/python3.10/site-packages/prophet/make_holidays.py", line 14, in <module>
>     import prophet.hdays as hdays_part2
>   File "/usr/local/lib/python3.10/site-packages/prophet/hdays.py", line 779, in <module>
>     class TU(Turkey):
>   File "/usr/local/lib/python3.10/site-packages/holidays/registry.py", line 178, in __init__
>     super().__init__(*args, **kwargs)
> TypeError: object.__init__() takes exactly one argument (the instance to initialize)
> ```
> 
> Please advise! We've pinned it for now.

I agree... the same issue is happening here when running 'import prophet'

hdays.py line 779 'class TU(Turkey): pass' does not work.

The holidays package does not appear to have changed, so I am not sure how this worked before... 
I tested installing the holidays package myself locally, and the following does not work

from holidays import Turkey

class TU(Turkey): pass

we can define a class Turkey from HolidayBase... for example the following works:

class Turkey(HolidayBase): pass # <---- still need to keep the details here for Turkey's holidays. this is just an example 
class TU(Turkey): pass


please fix this asap because prophet is being used in quite a few applications

**Comment by pjbhaumik** [comment]
the solution from @arkid15r works

please approve

**Comment by ppruthi** [comment]
Can you please fix this ASAP as this is breaking our CI pipelines 🙏

**Comment by thinkall** [comment]
Encountering the same issue. A quick fix and a new release would be super helpful. Thanks.

**Comment by k-rush** [comment]
+1 please prioritize merging the fix as this is it breaks an important scenario.

**Comment by tcuongd** [comment]
Thanks all for the report and @arkid15r for the quick fix, prophet 1.1.3 is now on PyPI and shouldn't have this issue.
